functions.js

```diff
@@ -1,5 +1,29 @@
-const firstFunction = () => {
-  return 'Hello';
+const add = (a, b) => {
+  return a + b;
 };
 
-firstFunction();
+const subtract = (a, b) => {
+  let result;
+  result = a - b;
+  return result;
+};
+
+function multiply(a, b) {
+  return a * b;
+}
+
+const divide_numbers = (a, b) => {
+  return a / d;
```

Comment:
Typo here - your are using an undefined `d` instead of `b`. This will cause a reference error!